source/_docs/guides/jenkins.md

```diff
@@ -0,0 +1,310 @@
+
+In this guide, we'll configure your existing Jenkins server to automatically test and deploy Drupal 8 to a Multidev environment on Pantheon following a code push to your GitHub repository. Jenkins will run tests against both the pull request and Pantheon's master branch and display the results.
+
+## Before You Begin
+
+You will need:
+
+- Root access to a server running Jenkins.
+- Install the following applications, both locally and on the Jenkins server. Verify the default Jenkins user has the ability to run them on the Jenkins server from the command line.
+    - [Git](https://git-scm.com/)
+    - [Composer](https://getcomposer.org/)
+    - [Drush](http://docs.drush.org/en/8.x/install-alternative)
+    - [Terminus](https://github.com/pantheon-systems/terminus#installing-with-composer)
+    - [Terminus Build Tools plugin](https://github.com/pantheon-systems/terminus-build-tools-plugin)
+
+    Verify you can run Terminus, Drush and Composer commands as the Jenkins user. Note that Terminus and Drush will need to be accessible from standard `PATH` directories.
+- Be sure the default Jenkins user has a [private key](https://stackoverflow.com/questions/37331571/how-to-setup-ssh-keys-for-jenkins-to-publish-via-ssh) in its `$HOME/.ssh` directory, and a user with the matching public key installed on Pantheon. Clone a Pantheon site to the Jenkins user's home directory to test.
+- The Jenkins user's git user name and email mail should be set in **Manage Jenkins**, **Configure System**
+- In the Jenkins user's $HOME/.ssh/config (if this file doesn't exist, create it), add these two lines:
+
+    ```
+     Host *
+         StrictHostKeyChecking no
+    ```
+
+- The following Jenkins plugins should be enabled: [Git](https://wiki.jenkins-ci.org/display/JENKINS/Github+Plugin) , [Github](https://wiki.jenkins-ci.org/display/JENKINS/GitHub+Plugin), [Github Pull Request Builder](https://wiki.jenkins-ci.org/display/JENKINS/GitHub+pull+request+builder+plugin), [Environment Injector](https://wiki.jenkins-ci.org/display/JENKINS/EnvInject+Plugin), [Conditional Build Step](https://wiki.jenkins-ci.org/display/JENKINS/Conditional+BuildStep+Plugin), [Run Condition](https://wiki.jenkins-ci.org/display/JENKINS/Run+Condition+Plugin), and [Rebuild](https://wiki.jenkins.io/display/JENKINS/Rebuild+Plugin).
+- We recommended using matrix-based security when using Jenkins. Anonymous users should have read access to Jenkins projects.
+- Ability to create a Drupal site on Pantheon, with [Multidev](/docs/multidev/) enabled.
+- A [GitHub](https://github.com) account.
+
+## Create A New Site
```

Comment:
That's a good question for anyone using our GH project, probably a separate doc in itself. I can add a sentence that should explain it generally.